src/main/java/de/blackforestsolutions/dravelopsdatamodel/Price.java

```diff
@@ -1,29 +1,82 @@
 package de.blackforestsolutions.dravelopsdatamodel;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.hazelcast.nio.serialization.Portable;
+import com.hazelcast.nio.serialization.PortableReader;
+import com.hazelcast.nio.serialization.PortableWriter;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.experimental.Accessors;
 
+import java.io.IOException;
+import java.io.Serializable;
 import java.util.Currency;
 
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.DravelOpsPortableFactory.DRAVEL_OPS_FACTORY_ID;
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.DravelOpsPortableFactory.PRICE_CLASS_ID;
+import static de.blackforestsolutions.dravelopsdatamodel.hazelcast.classdefinition.PriceClassDefinition.*;
+
 @Getter
 @JsonDeserialize(builder = Price.PriceBuilder.class)
-public final class Price {
+public final class Price implements Serializable, Portable {
+
+    private static final long serialVersionUID = -3355934959094849304L;
 
-    private final PriceType priceType;
+    private PriceType priceType;
 
-    private final Currency currencyCode;
+    private Currency currencyCode;
 
-    private final Long smallestCurrencyValue;
+    private Long smallestCurrencyValue;
```

Comment:
Maybe we need this to implement Portable, but we arent immutable anymore in this case. How do we enforce to not use setter for this method?

Reply:
Done